src/pkg/aws/ecs/logs.go

```diff
@@ -132,28 +132,50 @@ func TailLogGroups(ctx context.Context, logGroups ...string) (EventStream, error
 	return &cs, nil
 }
 
-func TailLogGroup(ctx context.Context, logGroupArn string, logStreams ...string) (EventStream, error) {
+// LogGroupInput is like cloudwatchlogs.StartLiveTailInput but with only one loggroup and one logstream prefix.
+type LogGroupInput struct {
+	LogGroup              string
+	LogStreamNames        []string
+	LogStreamNamePrefix   string
+	LogEventFilterPattern string
+}
+
+func TailLogGroup(ctx context.Context, input LogGroupInput) (EventStream, error) {
+	var pattern *string
+	if input.LogEventFilterPattern != "" {
+		pattern = &input.LogEventFilterPattern
+	}
+	var prefixes []string
+	if input.LogStreamNamePrefix != "" {
+		prefixes = []string{input.LogStreamNamePrefix}
+	}
 	return startTail(ctx, &cloudwatchlogs.StartLiveTailInput{
-		LogGroupIdentifiers: []string{getLogGroupIdentifier(logGroupArn)},
-		LogStreamNames:      logStreams,
+		LogGroupIdentifiers:   []string{getLogGroupIdentifier(input.LogGroup)},
+		LogStreamNames:        input.LogStreamNames,
+		LogStreamNamePrefixes: prefixes,
+		LogEventFilterPattern: pattern,
 	})
 }
 
-func Query(ctx context.Context, logGroupArn string, start time.Time, end time.Time) ([]LogEvent, error) {
-	region := region.FromArn(logGroupArn)
+func Query(ctx context.Context, lgi LogGroupInput, start time.Time, end time.Time) ([]LogEvent, error) {
+	region := region.FromArn(lgi.LogGroup) // FIXME: ensure it's a log group ARN
```

Comment:
I would rather either:
- Change LogGroup to LogGroupARN and require it to be an ARN
- Or, Detect if it is an ARN, if not, use the default region to create the config, since we are already requiring a region to be supplied else where.

Reply:
It's always an ARN; renamed.